### DISCUSSION TITLE
百度网盘恢复挂载失败

### DISCUSSION THREAD

**Discussion by FFFGGL** [AlistGo/alist#3508] [Q&A]
![image](https://user-images.githubusercontent.com/37217571/219864446-d568ec08-e269-4245-92aa-e694e8bf2772.png)
![image](https://user-images.githubusercontent.com/37217571/219864477-6c8a9ba2-d679-461a-8c84-992636191360.png)
出现这种问题怎么办！！

**Comment by goodrabbit** [comment]
貌似作者更改了API的回调地址导致的。研究了一下，正确获得令牌的链接应该是这个：

https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://alist.nn.ci/tool/baidu/callback&scope=basic,netdisk&qrcode=1

![屏幕截图 2023-02-21 182552](https://user-images.githubusercontent.com/5088561/220319859-71abe3dc-fd6d-40e0-b862-a692c7a42872.jpg)

**Comment by FFFGGL** [comment]
欧克·！感谢！我去试试！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/alist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月21日(星期二) 晚上6:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/alist] 百度网盘恢复挂载失败 (Discussion #3508)





 
貌似作者更改了API的回调地址导致的。研究了一下，正确获得令牌的链接应该是这个：
 
https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&amp;client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&amp;redirect_uri=https://alist.nn.ci/tool/baidu/callback&amp;scope=basic,netdisk&amp;qrcode=1
